src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/Enrich.scala

```diff
@@ -184,7 +185,7 @@ object Enrich {
     resolver: Json,
     enrichmentConfs: List[EnrichmentConf],
     cachedFiles: DistCache[List[Either[String, String]]]
-  ): SCollection[Validated[List[String], EnrichedEvent]] =
+  ): SCollection[Validated[String, EnrichedEvent]] =
```

Comment:
```suggestion
  ): SCollection[Validated[BadRow, EnrichedEvent]] =
```

#21 is not taken care of